src/Workleap.DomainEventPropagation.Abstractions/DomainEventAttribute.cs

```diff
@@ -3,10 +3,6 @@
 [AttributeUsage(AttributeTargets.Class, Inherited = false)]
 public sealed class DomainEventAttribute : Attribute
 {
-    public DomainEventAttribute(string name) : this(name, EventSchema.EventGridEvent)
-    {
-    }
```

Comment:
This was redundant, given we already have the other constructor with the default parameter.

Reply:
Removing this member is a binary breaking change. Not sure this is intended.

Reply:
@meziantou should we then: 
1. keep the original constructor
2. provide a second constructor, but remove the default value for the `eventSchema` constructor parameter?

This would no longer be breaking, but please advise if I didn't get this right.

Reply:
You should keep both constructors. The second ctor can have the optional parameter if you want, but in this case it won't matter as there are only 1 new parameter and the first one is not optional.

This way the previous ctor still exists. So, if someone compiles a lib against a previous version of `DomainEventAttribute`, you can still use this lib when your application targets the new version of the NuGet.

Reply:
In the end, following a discussion I had with @meziantou, we have decided to keep the constructor, but make the `eventSchema` parameter mandatory and its default value.